### PR TITLE
Add converters

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: vividus/src/integrationTest/java/org/vividus/configuration/Bean.java [hunk 1]
```diff
@@ -14,29 +14,32 @@
...
+import java.lang.System.Logger.Level;
 import java.util.Locale;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
 
-import org.junit.jupiter.api.Test;
-
-class StringToLocaleConverterTests
+public class Bean
```
Comment (uarlouski) [inline vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: why is it called bean?

Comment (ikalinin1) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: any concerns?

Comment (uarlouski) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: These guys have lots of concerns
Painted_Pony_Bean

Comment (uarlouski) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: if seriously this name says nothing about mission of this class

Comment (ikalinin1) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: This is just a bean

Comment (ikalinin1) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: I can use your name if you have a good one, am I?

Comment (uarlouski) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: Name of class should give some idea about why this class is needed https://www.oracle.com/technetwork/java/codeconventions-135099.html

Comment (ikalinin1) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: So, what's yotut pick for pojo with fields?

Comment (uarlouski) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: at least the name should reflect somehow that this bean holds fields set to values passed through spring **converters**

File: vividus/src/main/java/org/vividus/spring/StringToEditorGenericConverter.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    @SuppressWarnings("unchecked")
+    public void init()
+    {
+        Field findField = ReflectionUtils.findField(PropertyEditorRegistrySupport.class, "defaultEditors");
+        findField.setAccessible(true);
+        defaultEditors = (Map<Class<?>, PropertyEditor>) ReflectionUtils.getField(findField, propertyEditorRegistry);
+    }
+
+    public void setPropertyEditorRegistry(PropertyEditorRegistrySupport propertyEditorRegistry)
```
Comment (uarlouski) [inline vividus/src/main/java/org/vividus/spring/StringToEditorGenericConverter.java]: is it correct place for setter?

File: vividus/src/integrationTest/java/org/vividus/configuration/Bean.java (hunk 1 above)
Comment (ikalinin1) [reply vividus/src/integrationTest/java/org/vividus/configuration/Bean.java]: Like any other bean covered for us by spring :)

**Comment by valfirst** [comment]
Waiting for decision regarding https://github.com/spring-projects/spring-framework/pull/24311

**Comment by valfirst** [comment]
Keep it open until PR to Spring is not resolved

**Comment by valfirst** [comment]
Superseded by other pull requests

**Pull request closed without merging**